tooling/minitest/src/ub/print.rs

```diff
@@ -0,0 +1,31 @@
+use crate::*;
+
+#[test]
+fn print_success() {
+    let locals = [];
+
+    let b0 = block!(
+        print(const_int::<u32>(42), 1), // ints can be printed
+    );
+    let b1 = block!(exit());
+
+    let f = function(Ret::No, 0, &locals, &[b0, b1]);
+    let p = program(&[f]);
+    dump_program(p);
+    assert_stop(p);
```

Comment:
Ideally, this would test that we got the right output. But I assume that's non-trivial? In that case we should open an issue to track this.

Reply:
Ideally whoever sets up the interpreter can provide a `Box<dyn Writer>` that `print` will print to, or something like that. Then one can use stdout or one can use a `Vec<u8>` and check the result.

But anyway, I'm okay with not making this part of this PR.